warning: missing initializer for member 'esp_vfs_littlefs_conf_t::partition' [-Wmissing-field-initializers]

### Board

ESP32 Dev Module

### Device Description

Plain Module

### Hardware Configuration

Nothing Attached

### Version

v2.0.14

### IDE Name

PlatformIO

### Operating System

Windows 11

### Flash frequency

N/A

### PSRAM enabled

no

### Upload speed

N/A

### Description

When compiling with more strict gcc configurations (`-Wall -Wextra -Wunused -Wmisleading-indentation -Wduplicated-cond -Wlogical-op -Wnull-dereference`) I get the following warnings (or errors when compiling with -Werror)

It seems that the used IDF extended the `esp_vfs_littlefs_conf_t` type which is now not  fully initialized.

```
C:/Users/Thomas/.platformio/packages/framework-arduinoespressif32/libraries/LittleFS/src/LittleFS.cpp: In member function 'bool fs::LittleFSFS::begin(bool, const char*, uint8_t, const char*)':
C:/Users/Thomas/.platformio/packages/framework-arduinoespressif32/libraries/LittleFS/src/LittleFS.cpp:86:5: warning: missing initializer for member 'esp_vfs_littlefs_conf_t::partition' [-Wmissing-field-initializers]      
     };
     ^
C:/Users/Thomas/.platformio/packages/framework-arduinoespressif32/libraries/LittleFS/src/LittleFS.cpp:86:5: warning: missing initializer for member 'esp_vfs_littlefs_conf_t::read_only' [-Wmissing-field-initializers]      
C:/Users/Thomas/.platformio/packages/framework-arduinoespressif32/libraries/LittleFS/src/LittleFS.cpp:86:5: warning: missing initializer for member 'esp_vfs_littlefs_conf_t::grow_on_mount' [-Wmissing-field-initializers]
```

### Sketch

```cpp
N/A
```


### Debug Message

```plain
N/A
```


### Other Steps to Reproduce

N/A

### I have checked existing issues, online documentation and the Troubleshooting Guide

- [X] I confirm I have checked existing issues, online documentation and Troubleshooting guide.

--- a/libraries/LittleFS/src/LittleFS.cpp
+++ b/libraries/LittleFS/src/LittleFS.cpp
@@ -81,7 +81,9 @@ bool LittleFSFS::begin(bool formatOnFail, const char * basePath, uint8_t maxOpen
     esp_vfs_littlefs_conf_t conf = {
       .base_path = basePath,
       .partition_label = partitionLabel_,
+      .partition = NULL,
       .format_if_mount_failed = false,
+      .read_only = false,
       .dont_mount = false,
       .grow_on_mount = true
     };

